book/website/afterword/subprojects.md

```diff
@@ -88,21 +156,3 @@ The next chair rotation is scheduled for September 2024.
 **Maintainers**
 - [Julien Colomb](https://book.the-turing-way.org/afterword/contributors-record#julien-colomb)
 - [Malvika Sharan](https://book.the-turing-way.org/afterword/contributors-record#malvika-sharan)
-
-## Informal Initiatives
```

Comment:
@PatriLoto - via our collaborations in the Accessibility Working Group, maybe Metadocencia can be added here too? Maybe in collaborating projects rather than this initiatives one?